generators/client/templates/angular/webpack/_prettier.js

```diff
@@ -0,0 +1,22 @@
+const prettier = require('prettier');
+const fs = require('fs');
+
+const walkSync = function (dir, filelist) {
```

Comment:
Isnt it possible to do the same using prettier CLI directly without this script?